YAXLib/MemberWrapper.cs

```diff
@@ -898,6 +900,10 @@ private void ProcessYaxAttribute(object attr)
             {
                 IsAttributedAsDontSerializeIfNull = true;
             }
+            else if (attr is YAXElementOrder)
+            {
+                Order = (attr as YAXElementOrder).Order;

```

Comment:
"This method performs a stable sort; that is, if the keys of two elements are equal, the order of the elements is preserved."

https://msdn.microsoft.com/en-us/library/bb549422%28v=vs.100%29.aspx